Header header
string ip_address
string device_name
ros_mbed_bridge/KeyValue[] values